src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -166,6 +168,11 @@ class BtreeLBAMapping : public BtreeNodeMapping<laddr_t, paddr_t> {
     return p.modified_since(parent_modifications);
   }
 
+  void maybe_fix_pos() final {
+    assert(is_parent_valid());
+    auto &p = static_cast<LBALeafNode&>(*parent);
+    p.maybe_fix_mapping_pos(*this);
```

Comment:
Since there is `parent_modified()` mechanism, looks we can skip it (lookup the key-pos again) if the parent isn't modified?